addons/l10n_ch/data/account_vat2011_data.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
         <!--
-        #  TVA - Taxe sur la Valeur Ajoutée
+        #  VAT - Value added tax
         -->
         <record model="account.tax.template" id="vat_25">
-            <field name="name">TVA due a 2.5% (TR)</field>
+            <field name="name">2.5% Sales vat</field>
```

Comment:
VAT

Reply:
It is already a VAT context, is it relevant in this case to add an information twice ?